api/src/pcapi/admin/custom_views/venue_view.py

```diff
@@ -191,7 +215,9 @@ def update_model(self, new_venue_form: Form, venue: Venue) -> bool:
         if changed_attributes:
             search.async_index_offers_of_venue_ids([venue.id])
 
-        update_external_pro(venue.bookingEmail)
+        # No need to update both former and new bookingEmail, which can not be changed from backoffice
+        for email in _get_emails_by_venue(venue):
+            update_external_pro(email)
```

Comment:
En tant que dev qui ne connait pas ce sujet, j'ai du mal à comprendre pourquoi il y a ce `for`, malgré le commentaire :grimacing: 
C'est peut être parce que je manque de contexte, ou parce que le contexte devrait être dans le commentaire ?